src/index.test.ts.md

```diff
@@ -388,7 +383,6 @@ Generated by [AVA](https://avajs.dev).
       ␊
       contract StringConstructorUpgradeable is Initializable {␊
           function __StringConstructor_init(string memory message) internal onlyInitializing {␊
-              __StringConstructor_init_unchained(message);␊
```

Comment:
This is going to reintroduce unused parameter warnings. Notice how `message` is no longer used in the function body.